demo/site/src/common/blocks/RichTextBlock.tsx

```diff
@@ -75,14 +75,17 @@ const defaultRichTextRenderers: Renderers = {
      */
     entities: {
         // key is the entity key value from raw
-        LINK: (children, data: LinkBlockData, { key }) =>
-            isValidLink(data) ? (
-                <InlineLink key={key} data={data}>
+        LINK: (children, data: LinkBlockData, { key }) => {
+            const mergedKey = key + children?.toString();
```

Comment:
Is stringifying the children a good idea? Does this even work in all scenarios?

Reply:
can't you create a random key?

Reply:
Should be possible with something like useConstant.

Reply:
> Is stringifying the children a good idea? Does this even work in all scenarios?

No, I just realised it doesn't work for already formatted strings because then we receive a React Node (which ist converted to [object Object]). 

> can't you create a random key?
> Should be possible with something like useConstant.

I tried saving it using `useRef`. However, when adding a new link or deleting an existing one, I get an error:

<img width="570" alt="Bildschirmfoto 2025-01-20 um 10 12 35" src="https://github.com/user-attachments/assets/b547c949-bde1-42df-9b67-321a8ab8d8b7" />

<img width="568" alt="Bildschirmfoto 2025-01-20 um 09 52 09" src="https://github.com/user-attachments/assets/0a5b052b-7f1c-4054-97dd-db99e4a1c333" />

Reply:
I now added a recursive function to extract the text from the react component (f9f69e6b4e60f52df37273843aba70eed485753d). 

This still leave the issue: When the first and second part of the link is identical (e.g., Tuk(-)Tuk). But as I said, I think this is neglectable.

Reply:
asking again: can't you create a random key?

Reply:
Here you go: https://github.com/vivid-planet/comet/pull/3258